.github/workflows/ingestion.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         run: npm ci
 
       - name: Ingest
-        run: npm run ingest -- --chatDomain ${{ inputs.chat-domain }} --wipeTuples ${{ inputs.wipe-tuples }}
+        run: npm run ingest -- --chatHost ${{ inputs.chat-domain }} --wipeTuples ${{ inputs.wipe-tuples }}
```

Comment:
Will need to change the default on line 9 as well.

Reply:
Good catch! I changed it up on line 9.